CollapsibleExample/ExampleSubclassFilterViewController.m

```diff
@@ -33,44 +33,21 @@ - (void)didReceiveMemoryWarning {
 }
 
 - (void)createManagersAndPopulateData{
```

Comment:
I did have that at one point, though if it's in the init it forces the data to be set earlier instead of giving the developer subclassing the option to put it wherever they want (after executing other code)
This method just exists empty in the filterviewcontroller. Do you suggest forcing it in the init instead of giving the option to set the data in other parts?